src/forms-opportunities/handler.js

```diff
@@ -90,6 +195,7 @@ export async function processOpportunityStep(context) {
   await createLowNavigationOpportunities(finalUrl, latestAudit, scrapedData, context, excludeForms);
   await createLowViewsOpportunities(finalUrl, latestAudit, scrapedData, context, excludeForms);
   await createLowConversionOpportunities(finalUrl, latestAudit, scrapedData, context, excludeForms);
+  await sendA11yIssuesToMystique(latestAudit, scrapedData, context);
```

Comment:
`createAccessibilityOpportunity` as this flow will just detect the a11y issues, create an a11y oppoty and also, push the issues to mystique.

Reply:
For now, it is creating opportunity, but once Mystique integration will be available then it will only send issues to Mystique hence `sendA11yIssuesToMystique`

Reply:
But we'll create the oppoty in this flow and then push it to mystique to be consistent with other oppoty.
then we'll write a handler to consume M's response and update the oppoty.